migtests/scripts/functions.sh

```diff
@@ -230,17 +230,38 @@ run_sqlplus() {
 }
 
 export_schema() {
-	args="--export-dir ${EXPORT_DIR}
-		--source-db-type ${SOURCE_DB_TYPE}
-		--source-db-user ${SOURCE_DB_USER}
-		--source-db-password ${SOURCE_DB_PASSWORD}
+    # Default values
+    export_dir="${EXPORT_DIR}"
+    source_db_schema="${SOURCE_DB_SCHEMA}"
+
+    # Process arguments
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            export_dir)
+                export_dir="$2"
+                shift 2
+                ;;
+            source_db_schema)
+                source_db_schema="$2"
+                shift 2
+                ;;
+            *)
+                break
+                ;;
+        esac
+    done
```

Comment:
this is new addition, i hope you have verified other run scripts also which use export_schema command